packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -24,6 +24,8 @@ export function extractJsonObject(str: string): any {
 // To get around this, fs.realpathSync.native() is called to get the
 // URI with the actual file name.
 export function flushFilePath(filePath: string): string {
+  // filePath is in the format of "/Users/{user-name}/{path-to-apex-file.cls}" on Mac OSX
+  // filePath is in the format of "c:\\Users\\{user-name}\\{path-to-apex-file.cls}" on Windows
```

Comment:
Randi ran this branch on her Windows machine this morning.  The code ran w/o any additional changes needed, and I added a comment here of what the format of `filePath` is here on Windows.

Reply:
I probably wasn't clear in my original comment.  The issue with this comment is the path isn't guaranteed to be in the users home space on either windows or mac.  `/Users/{user-name}/` and `\\Users\\{user-name}\\` aren't always present.  Is the point of this comment to communicate that the filePath string will not include the file:/ scheme?  If so I could see going with something like ```
// The filePath should be an fs filepath and not a VSCode URI string. No scheme (file:/) should be present.
// Examples: 
// "/Users/{user-name}/{path-to-apex-file.cls}" on Mac OSX
// "c:\\Users\\{user-name}\\{path-to-apex-file.cls}" on Windows```